### DISCUSSION TITLE
Jetpack Compose Experimentation

### DISCUSSION THREAD

**Discussion by Alton09** [twilio/twilio-video-app-android#196] [General]
As a learning exercise, we are converting xml layouts to Kotlin via Jetpack Compose. Feel free to provide feedback on this branch or provide any pull requests to expand on this idea.

https://github.com/twilio/twilio-video-app-android/tree/jetpack-compose-spike

**Comment by dinhthaidaica** [comment]
@Alton09 Any guidance about how to use this experimentation?
